.github/actions/e2e/setup-cluster/action.yaml

```diff
@@ -98,6 +99,7 @@ runs:
           github.com/run-url: "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
           testing/type: "e2e"
           testing/cluster: "$CLUSTER_NAME"
+          test/git_ref: "$GIT_REF"
```

Comment:
Should any other resources that we create separately be tagged like this? OIDC providers, etc?

Reply:
Add OICD tag and I believe all other resources are created and tagged by EKSCTL